src/core/cluster/cluster_service.go

```diff
@@ -115,7 +117,7 @@ func (c *ClusterServer) UpdateFileContentsCS(ctx context.Context,
 	request *hootpb.UpdateFileContentsCSRequest) (*hootpb.UpdateFileContentsCSResponse, error) {
 	file_id, err := uuid.FromBytes(request.FileId.Value)
 	if err != nil {
-		return nil, ErrInvalidId
+		return nil, status.Error(codes.InvalidArgument, ErrInvalidId.Error())
```

Comment:
Is keeping the 1st return value nil intentional here, or was this an oversight?

Reply:
That was an oversight. I'll fix this.

Reply:
Done.